### DISCUSSION TITLE
小玉換臉事件—deepfak

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#733] [General]
前陣子鬧得沸沸揚揚的的小玉事件，讓大家正視換臉技術和隱私權相關的問題。
Deepfake是個雙面刃，由於使用者動機、目的的不同，不能斷定deepfake就是有害隱私權的工具。
所謂知己知彼，百戰百勝，若想好好防禦deepfake帶來的影響，我們勢必要更了解deepfake的運作與功能。
https://pansci.asia/archives/342432

**Comment by Valentinaa0214** [comment]
標題少了一個E！這件事剛被爆出來時真的是讓人十分憤怒，科技本是幫助人類進步的工具，但卻被有心人士拿來做這麼不當的利用，讓人不免擔心AI的進步是否會因為不同的使用方式與動機，而成為一把雙面刃。
